experimental/traffic-portal/src/app/api/server.service.ts

```diff
@@ -186,4 +187,33 @@ export class ServerService extends APIService {
 
 		return this.put(`servers/${id}/status`, {offlineReason, status}).toPromise();
 	}
+
+	/**
+	 * Creating new Status.
+	 *
+	 * @param payload containes name and description for the status.
+	 * @returns The 'response' property of the TO status response. See TO API docs.
+	 */
+	public async createStatus(payload: RequestStatus): Promise<ResponseStatus> {
+		return this.post<ResponseStatus>("statuses", payload).toPromise();
+	}
+
+	/**
+	 * Updates status Details.
+	 *
+	 * @param payload containes name and description for the status., unique identifier thereof.
```

Comment:
same as above

also, though, has an extra phrase: `, unique identifier thereof.` which doesn't seem to belong there.